tests/fsharp/core/printing/output.1000.stdout.bsl

```diff
@@ -2765,7 +2765,7 @@ val ShortName: string = "hi"
 > val list2: int list = [1]
 
 module FSI_0317.
-       C6f6ae524efb4d95b2b2eaa363022f9d4a28c777f788498ca81a55b9ec1aad1a
+       A8a951db8294f99e95ae1d276a7ddaefd93d1548e6bf749bdeae55d2649682b3
```

Comment:
Is this constant name defined somewhere, to ensure it is stable?

Reply:
It's not stable.  It's generated miserably ... I may get around to see if it can be made deterministic though.